generator/uscore/uscore_generator.rb

```diff
@@ -703,6 +703,10 @@ def create_resource_profile_test(sequence)
         bindings = sequence[:bindings]
           .select { |binding_def| ['required', 'extensible'].include? binding_def[:strength] }
 
+        bindings.each do |binding|
+          binding[:path].gsub!('.class', '.local_class')
+          binding[:path] = 'local_class' if binding[:path] == 'class'
```

Comment:
Only one of these should be necessary.

Reply:
fixed. Replaced with a better gsub expression